src/core/remote.js

```diff
@@ -1610,7 +1610,7 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
   const request = new Request(this, 'book_offers');
 
   request.message.taker_gets = {
-    currency: Currency.json_rewrite(gets.currency, {force_hex: true})
```

Comment:
I think this is a bug in the currency class, I created this ticket: https://ripplelabs.atlassian.net/browse/RLJS-403

I'm going to revert this change to avoid scope creep on this pull request.